lib/index.js

```diff
@@ -36,6 +36,10 @@ Plotly.register([
 
     require('./scattermapbox'),
 
+    require('./carpet'),
+    require('./scattercarpet'),
+    require('./contourcarpet'),
```

Comment:
Maybe carpet traces should be part of the [`cartesian`](https://github.com/plotly/plotly.js/blob/master/lib/index-cartesian.js) bundle?

If `scatterternary` wasn't already in there I'd vote for _no_. But now that I realise `scatteternary` is part of the cartesian bundle, I'm not so sure.